packages/agoric-cli/package.json

```
@@ -8,6 +8,11 @@
     "agoric": "src/entrypoint.js",
     "agops": "src/bin-agops.js"
   },
+  "exports": {
+    "./src/entrypoint.js": "./src/entrypoint.js",
+    "./lib/helpers.js": "./lib/helpers.js",
+    "./src/helpers.js": "./src/helpers.js"
```

The `lib` variant no longer exists. It's only referenced by the loadgen for compat with older SDK revisions.
```suggestion
    "./src/helpers.js": "./src/helpers.js"
```